presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -52,106 +52,132 @@ PeriodicTaskManager::PeriodicTaskManager(
     : driverCPUExecutor_(driverCPUExecutor),
       httpExecutor_(httpExecutor),
       taskManager_(taskManager),
-      memoryManager_(velox::memory::MemoryManager::getInstance()),
-      connectors_(connectors) {}
+      memoryAllocator_(velox::memory::MemoryAllocator::getInstance()),
```

Comment:
I think Presto server can pass cache_ object here directly?

Reply:
I'm trying to limit the passed in entities. Here we can directly get from global getter so I think it can make the interface cleaner.

Reply:
I think it is better to explicitly pass those entities from presto server instead of cast to cache object internally. This looks more hack to me. Thanks!